csrc/optimization/pre_segmenter.cpp

```diff
@@ -21,6 +22,7 @@ void PreSegmenter::runPass(Fusion* fusion) {
   OptimizationPass<ConsecutiveCastPass>::runPass(fusion);
   OptimizationPass<AddAxiomsPass>::runPass(fusion);
   OptimizationPass<MarkAliasesPreparePass>::runPass(fusion);
+  OptimizationPass<ConcretizeSymbolicRootDomainPass>::runPass(fusion);
```

Comment:
About the name of this pass: the term "concretize" is already quite overloaded, referring to concretizing symbolic `IterDomain`s as well as determining the origin of broadcast domains at lowering. Also in case you agree with my suggestion above to replace with symbolic extents as well, it might no longer be meaningful to say we are concretizing. Maybe something like `ExactMappedExtentStandardizationPass` would be clearer?

Reply:
Sounds good. I don't quite get the meaning of `Standardization` and `Substitution` seems more direct as we are replacing non-const extents with const extents. So, what about `ExactMappedExtentSubstitutionPass`?

Reply:
Sounds good to me.